tests/test_docs.py

```diff
@@ -6,30 +6,40 @@
 import os
 import subprocess
 import unittest
+import platform
 
 
 class TestDoc(unittest.TestCase):
 
     @property
     def path_to_docs(self):
-        dirname, filename = os.path.split(os.path.abspath(__file__))
-        return os.path.sep.join(dirname.split(os.path.sep)[:-1] + ['docs'])
+        dirname, file_name = os.path.split(os.path.abspath(__file__))
+        return dirname.split(os.path.sep)[:-1] + ["docs"]
 
     def test_html(self):
         wd = os.getcwd()
-        os.chdir(self.path_to_docs)
+        os.chdir(os.path.sep.join(self.path_to_docs))
+
+        if platform.system() != "Windows":
+            response = subprocess.run(["make", "html"])
+            self.assertTrue(response.returncode == 0)
+        else:
+            response = subprocess.call(["make", "html"], shell=True)
+            self.assertTrue(response == 0)
 
-        response = subprocess.run(["make", "html"])
```

Comment:
Were the changes in this file actually a necessary, or was the doc build failing because they were simply failing? from what I've read:
`subprocess.call(...)` is the same as `subprocess.run(...).returncode`. It boils down to then, Why would windows require the `shell=True` kwarg, and is that returning an erroneous pass?